src/scripts/docs/build.sh

```diff
@@ -23,9 +23,12 @@ swagger validate "${SWAG_SPEC_LOCATION}"
 
 npx widdershins@3.6.7 -u docs/.widdershins/templates -e docs/.widdershins/config.json "$SWAG_SPEC_LOCATION" -o ./docs/docs/reference/api.mdx
 
-node ./docs/scripts/gen-faq.js
-node ./docs/scripts/fix-api.js ./docs/docs/reference/api.mdx
-node ./docs/scripts/config.js docs/config.js
+# node ./docs/scripts/gen-faq.js
+# node ./docs/scripts/fix-api.js ./docs/docs/reference/api.mdx
+# node ./docs/scripts/config.js docs/config.js
+# replace with npm run gen, runs in projects with /docs
+
+npm run gen
```

Comment:
Pretty sure this won't work, because you're not in the `docs` folder. Did you test this code?

Reply:
True it would only work if you are in `docs`.
Can we cd into it or is that not an option?

If not we can keep it like it was before.

Reply:
yes, it must look like the command with the (cd docs; npm i) before

Reply:
Ok added it now to run in sub shell.
I tested in `/kratos` and worked fine.